src/Symfony/Component/Encryption/Ciphertext.php

```diff
@@ -0,0 +1,170 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Encryption;
+
+use Symfony\Component\Encryption\Exception\DecryptionException;
+use Symfony\Component\Encryption\Exception\MalformedCipherException;
+
+/**
+ * This class is responsible over the payload API.
+ *
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ *
+ * @experimental in 5.3
+ *
+ * @internal
+ */
+class Ciphertext
+{
+    /**
+     * @var string algorithm for the encryption
+     */
+    private $algorithm;
+
+    /**
+     * @var string the encoded payload
+     */
+    private $payload;
+
+    /**
+     * @var string nonce for the algorithm
+     */
+    private $nonce;
+
+    /**
+     * @var array additional headers
```

Comment:
should you document that it is a key->value array?
related to the `getHeader()`